### PR TITLE
`shed.yml` cleanups

### PR DESCRIPTION
**Pull request by bgruening**
@erasche here are my last changes! Thanks for pushing me :)

### CODE REVIEW

File: packages/package_mummer_3_23/.shed.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 long_description: |
   This Galaxy ToolShed package downloads and installs the MUMmer
   package, for use by other Galaxy tools/wrappers.
+
+  http://mummer.sourceforge.net/
 name: package_mummer_3_23
 owner: iuc
 remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/packages/package_mummer_3_23
-type: unrestricted
+type: tool_dependency_definition
```
Comment (erasche) [inline packages/package_mummer_3_23/.shed.yml]: ooh this is going to be fun. Wonder how the TS will behave with this. Nice catch :)

File: packages/package_seqtk_1_0_r75/.shed.yml [hunk 2]
```diff
@@ -1,8 +1,11 @@
 categories:
 - Tool Dependency Packages
-description: seqtk package
+description: Contains a tool dependency definition that downloads and compiles version
+  0.1.0-r75 of the the segtk package.
```
Comment (erasche) [inline packages/package_seqtk_1_0_r75/.shed.yml]: We definitely need to standardise these, since we're basically copy+pasting a template.

**Comment by erasche** [comment]
LGTM, thanks for the post-merge cleanup!!

File: packages/package_mummer_3_23/.shed.yml (hunk 1 above)
Comment (bgruening) [inline packages/package_mummer_3_23/.shed.yml]: If there is only one file in the repository history changing the type should be possible.

Comment (martenson) [inline packages/package_mummer_3_23/.shed.yml]: I don't thing it is possible through the API :/

Comment (bgruening) [inline packages/package_mummer_3_23/.shed.yml]: Uh, API restrictions? But this is fixable without touching the TS internal code, this is good! :)

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/96/
Test (probably) passed

**Pull request merged**